layers/+lang/ruby/packages.el

```diff
@@ -65,6 +66,13 @@
     :init (spacemacs/add-to-hooks 'chruby-use-corresponding
                                   '(ruby-mode-hook enh-ruby-mode-hook))))
 
+(defun ruby/init-rufo ()
+  (use-package rufo
+    :init (dolist (mode '(ruby-mode enh-ruby-mode))
+            (add-hook (intern (concat (symbol-name mode) "-hook")) 'rufo-minor-mode)
+            (spacemacs/set-leader-keys-for-major-mode mode
+              "=b" 'rufo-format))))
+
```

Comment:
Why not add formatting for regions as well?

Reply:
AFAIK `rufo` works on the complete AST of the program, so it _may_ break something if we feed it to a narrowed region. Not sure if it's worth it. Also, the `rufo.el` package automatically formats on save, so there is no need for adding it to regions.

Reply:
Another thing: Use `SPC m=` -- which is what is used elsewhere.